plugins/dialog/plugin.js

```diff
@@ -2993,6 +2995,9 @@ CKEDITOR.DIALOG_STATE_BUSY = 2;
 	 * @param {String} dialogName The name of the dialog to open when executing
 	 * this command.
 	 * @param {Object} [ext] Additional command definition's properties.
+	 * You can provide additional property (`tab`) to ext object if you wish to open dialog on specific tab.
```

Comment:
You can add `tab` docs like:

```
@param {String} [ext.tab] Tab id which should be active when dialog is opened, e.g.
    here goes the code sample...
```

I also suggest changing `ext.tab` to `ext.tabId`.